client/src/components/resources/rides/RideAdd.jsx

```diff
@@ -39,6 +39,8 @@ export class RideAdd extends Component {
     e.preventDefault();
     const { uid } = this.props;
     const { driver, seats } = this.state;
+
+    // This should be an action
     this.props.firestore.add(
```

Comment:
Even though you are using firebase, still use actions. It gets confusing quickly if you are changing state directly in the components in some places and through actions in other places